.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
> If Watchman does not find the .watchmanconfig relative to your root directory, then it will simply load the watchman configuration present in /etc/watchman.json in your global configuration.

watchmanconfig can be safely deleted.